core/src/main/java/io/confluent/kafka/schemaregistry/masterelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -28,15 +30,7 @@
 
 import java.io.Closeable;
 import java.nio.ByteBuffer;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-
-import io.confluent.kafka.schemaregistry.storage.SchemaRegistryIdentity;
+import java.util.*;
```

Comment:
Maybe avoid star import?

Reply:
yeah, it's a checkstyle violation, too. I haven't properly configured my IDE for this project. Oops!